src/ripe_rainbow/interactive/drivers.py

```diff
@@ -203,6 +212,15 @@ def stop(self):
         self._flush_log()
         InteractiveDriver.stop(self)
 
+    def clear_cookies(self):
+        self.instance.delete_all_cookies()
+
+    def clear_local_storage(self):
+        self.instance.execute_script("localStorage.clear();")

```

Comment:
@gcandal this only clear the local storage for the current domain https://developer.mozilla.org/en-US/docs/Web/API/Storage/clear